.coveragerc

```diff
@@ -20,3 +20,6 @@ exclude_also =
 	# jaraco/skeleton#97
 	@overload
 	if TYPE_CHECKING:
+	# Coverage is only run on the latest Python version
+	# these will never be considered covered and we aim for 100% coverage
+	if sys\.version_info <
```

Comment:
Technically this matches `<= (3, 12)` but I think that's enough of an edge case, would require more complex regex and extra book-keeping when bumping the Python version, that it's not worth accounting for.

Reply:
This file is managed primarily in [skeleton](/jaraco/skeleton) and there's already an effort underway to [implement coverage aggregation](https://github.com/jaraco/skeleton/issues/130), which would obviate the need for this change.

Reply:
That would be better since then coverage would be correct in telling you that an environment-specific path was never taken.

Reply:
I'm glad you agree. I'll assume we should close this request, then, and await/pursue the upstream change. If you feel differently, feel free to advocate for including this change and we can re-open.